migrations/versions/0352_default_value_added_to_updated_at.py

```diff
@@ -0,0 +1,263 @@
+    op.drop_index('ix_service_inbound_api_updated_by_id', table_name='service_inbound_api')
+    op.drop_table('service_inbound_api')
+    op.alter_column('communication_items', 'name',
+               existing_type=sa.VARCHAR(),
+               type_=sa.Text(),
+               existing_nullable=False)
+    op.create_foreign_key(None, 'complaints', 'notification_history', ['notification_id'], ['id'])
+    op.alter_column('dm_datetime', 'month_name',
+               existing_type=sa.VARCHAR(),
+               type_=sa.Text(),
+               existing_nullable=False)
+    op.alter_column('dm_datetime', 'week_day_name',
+               existing_type=sa.VARCHAR(),
+               type_=sa.Text(),
+               existing_nullable=False)
+    op.alter_column('dm_datetime', 'quartal',
+               existing_type=sa.VARCHAR(),
+               type_=sa.Text(),
+               existing_nullable=False)
+    op.alter_column('dm_datetime', 'year_quartal',
+               existing_type=sa.VARCHAR(),
+               type_=sa.Text(),
+               existing_nullable=False)
+    op.alter_column('dm_datetime', 'year_month',
+               existing_type=sa.VARCHAR(),
+               type_=sa.Text(),
+               existing_nullable=False)
+    op.alter_column('dm_datetime', 'year_calendar_week',
+               existing_type=sa.VARCHAR(),
+               type_=sa.Text(),
+               existing_nullable=False)
+    op.create_index(op.f('ix_fido2_sessions_user_id'), 'fido2_sessions', ['user_id'], unique=True)
+    op.create_index(op.f('ix_inbound_numbers_service_id'), 'inbound_numbers', ['service_id'], unique=False)
+    op.alter_column('notification_history', 'notification_status',
+               existing_type=sa.TEXT(),
+               type_=sa.String(),
+               existing_nullable=True)
+    op.alter_column('notification_history', 'international',
+               existing_type=sa.BOOLEAN(),
+               nullable=False)
+    op.alter_column('notification_history', 'rate_multiplier',
+               existing_type=sa.NUMERIC(),
+               type_=sa.Float(),
+               existing_nullable=True)
+    op.alter_column('notifications', 'notification_status',
+               existing_type=sa.TEXT(),
+               type_=sa.String(),
+               existing_nullable=True)
+    op.alter_column('notifications', 'international',
+               existing_type=sa.BOOLEAN(),
+               nullable=False)
+    op.alter_column('notifications', 'rate_multiplier',
+               existing_type=sa.NUMERIC(),
+               type_=sa.Float(),
+               existing_nullable=True)
+    op.drop_index('ix_notifications_service_created_at', table_name='notifications')
+    op.alter_column('provider_details', 'updated_at',
+               existing_type=postgresql.TIMESTAMP(),
+               nullable=False)
+    op.alter_column('provider_details_history', 'updated_at',
+               existing_type=postgresql.TIMESTAMP(),
+               nullable=False)
```

Comment:
In addition to generating all these other changes, which you should delete manually (see [vanotify-team 224](https://github.com/department-of-veterans-affairs/vanotify-team/issues/224)), the migrate command didn't automatically include the defaults you specified.

Please see the [Alembic docs for alter_column](https://alembic.sqlalchemy.org/en/latest/ops.html?highlight=alter_column#alembic.operations.Operations.alter_column).  I think you need to provide a "server_default" value, which would just be datetime.now as in models.py.  Without this, applying the migration fails because nothing is assigned to the null values in existing rows, which precludes applying the migration in any of our AWS deployment environments.

I'm not sure how/if that is affecting your local db container.  I will pull this branch is see if I also encounter an error.